convert2rhel/redhatrelease.py

```diff
@@ -92,6 +97,22 @@ def _write_altered_yum_conf(self):
     def get_yum_conf_filepath():
         return YumConf._yum_conf_path
 
+    @staticmethod
+    def _is_modified():
+        """Return true if the YUM/DNF configuration file has been modified by the user."""
+        if pkgmanager.TYPE == 'yum':
+            conf = "/etc/yum.conf"
+        elif pkgmanager.TYPE == 'dnf':
+            conf = "/etc/dnf/dnf.conf"
+        else:
+            #raise NotImplementedError() return False corresponds better to the description
+            return False
```

Comment:
We can assume the system has either yum or dnf, otherwise we'd have issues already
```suggestion
        else:
            conf = "/etc/dnf/dnf.conf"
```